### PR TITLE
updating chat component version

### PR DESCRIPTION
**Pull request by josuhern**
After completing previous PR https://github.com/microsoft/omnichannel-chat-widget/pull/206
I am adding bumping version and Change_LOG

here is the final testing after all changes fixing the ([bug 3166695](https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3166695))
image

### CODE REVIEW

File: chat-widget/src/components/livechatwidget/common/defaultProps/dummyDefaultProps.ts [hunk 1]
```diff
@@ -400,7 +400,8 @@ export const dummyDefaultProps: ILiveChatWidgetProps = {
                 alignItems: "center",
                 margin: "0px 14px 0px 14px",
                 textOverflow: "ellipsis !important",
-                width: "max-content",
```
Comment (elopezanaya) [inline chat-widget/src/components/livechatwidget/common/defaultProps/dummyDefaultProps.ts:403]: is this related to the bump?, otherwise , add it to the changes log and include test evidence

**Review by elopezanaya — CHANGES_REQUESTED** [review]
about changes in dummyDefaults => 

is this related to the bump?, otherwise , add it to the changes log and include test evidence

File: chat-widget/src/components/livechatwidget/common/defaultProps/dummyDefaultProps.ts (hunk 1 above)
Comment (josuhern) [reply chat-widget/src/components/livechatwidget/common/defaultProps/dummyDefaultProps.ts:403]: That is related to this other PR https://github.com/microsoft/omnichannel-chat-widget/pull/206, yesterday Charlie told me that once I closed I had to bump the version and update the dummyDefualtProps

**Review by nareshme2016 — APPROVED** [review]

**Review by elopezanaya — APPROVED** [review]
